config/env_blank.hjson

```diff
@@ -29,9 +29,13 @@
     },
 
     "MIVIDEO": {
-        "service_account_json_filename": "umich-its-tl-reports-prod.json"
+        "default_last_timestamp": "2020-03-01T00:00:00+00:00",
+        "udp_service_account_json_filename": "umich-its-tl-reports-prod.json",
+        "kaltura_partner_id": -1,
+        "kaltura_user_secret": "kaltura_authentication_secret_here"
```

Comment:
Will the JSON schema validation give the user an error if they don't fill in an empty string?  It should.

If that validation works, and because the parameters are described in `README.md`, I could remove them.

Since HJSON allows for comments, how do you feel about using them here?

Reply:
Right now, it won't give an error with an empty string, but there are some ways to specify lengths: see [here](https://json-schema.org/understanding-json-schema/reference/string.html#length). We may want to open this as a separate issue. I just started with type checking the structure and variables and adding some enumerations and required properties.

I'm trying to keep the sample HJSON relatively uncluttered, and let people use comments in their own versions. That said, we may consider moving the field descriptions out of the `README.md` and into the HJSON at some point. I'd wait on it though, personally.

Reply:
Yeah, I noticed `minLength` in the documentation after I asked about it.  If it works right out of the box, I'll add that here.